astropy/coordinates/builtin_frames/heliocentric.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from ..representation import SphericalRepresentation
+from ..baseframe import (BaseCoordinateFrame, RepresentationMapping,
+                         TimeFrameAttribute)
+from .utils import DEFAULT_OBSTIME
+
+class Heliocentric(BaseCoordinateFrame):
```

Comment:
The axes of RA/Dec are aligned with the direction of ICRS. One possible name could be ```HCRS```, by analogy with ```GCRS```, which is a geocentric coordinate system with ```ICRS```-aligned axes.

As far as I know, ```HCRS``` is not used to mean anything else, so we could adopt it, though it might confuse people expecting it to have a formal definition like ```ICRS``` and ```GCRS``` and googling the term.